taskt/Core/Automation/Commands/Outlook Commands/SendOutlookEmailCommand.cs

```diff
@@ -1,93 +1,89 @@
-﻿using System;
-using System.Linq;
-using System.Xml.Serialization;
-using System.Data;
-using System.Windows.Forms;
+﻿using Microsoft.Office.Interop.Outlook;
+using System;
 using System.Collections.Generic;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
+using System.Windows.Forms;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
+using Application = Microsoft.Office.Interop.Outlook.Application;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Outlook Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to send emails with outlook")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to send emails with your currenty logged in outlook account")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Outlook Commands")]
+    [Description("This command sends emails with attachments in Outlook.")]
+
     public class SendOutlookEmailCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please Indicate Recipients (; delimited)")]
-        [Attributes.PropertyAttributes.InputSpecification("Enter the Email Addresses of the recipients in semicolon seperated values")]
-        [Attributes.PropertyAttributes.SampleUsage("test@test.com;test2@test.com")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyDescription("Recipient(s)")]
+        [InputSpecification("Enter the email address(es) of the recipient(s).")]
+        [SampleUsage("test@test.com;test2@test.com || {vEmail1};{vEmail2} || {vEmails}")]
```

Comment:
please provide a sample for a single recipient

Reply:
done